pandas/tools/plotting.py

```diff
@@ -1217,7 +1217,11 @@ def _adorn_subplots(self):
 
         if self.title:
             if self.subplots:
-                self.fig.suptitle(self.title)
+                if type(self.title) == list:
```

Comment:
Ok I'll use `is_list_like`. I purposely did not check the lengths as a feature. As it's written, if you only provide a list of two strings, but the plot has 3 suplots, the first two subplots will have a title and the third will be left without one.